src/tests/unit/tests/reports/components/__snapshots__/fast-pass-report-summary.test.tsx.snap

```diff
@@ -15,9 +15,9 @@ exports[`FastPassReportSummary renders per the snapshot 1`] = `
     }
     outcomeStats={
       Object {
-        "fail": 1,
+        "fail": NaN,
```

Comment:
Is this supposed to be `NaN`? It seems like we'd either want a number >= 0 always for this, right?